settings.py

```diff
@@ -0,0 +1,14 @@
+from pydantic.v1 import BaseSettings
+
+
+class Setting(BaseSettings):
+    PROJECT_NAME: str = "FastAPI City Temperature Management API"
+
+    DATABASE_URL: str | None = "sqlite+aiosqlite:///./city_temperature_management.db"
+
+    class Config:
+        case_sensitive = True
+        env_file = ".env"
```

Comment:
The `env_file` is set to `.env`, which is good for local development. Ensure that this file is included in your `.gitignore` to prevent sensitive information from being committed to version control.